misc/testerina/modules/testerina-core/src/main/ballerina/concurrentExecuter.bal

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2024 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+isolated function executeTestIso(TestFunction testFunction, DataProviderReturnType? testFunctionArgs) {
```

Comment:
```suggestion
isolated function executeIsolatedTest(TestFunction testFunction, DataProviderReturnType? testFunctionArgs) {
```

Reply:
Does not it sound like the test function which we are executing is isolated? What I need is a way to separate the naming between isolated function and not isolated counter part.